tools/ci-cdk/canary-lambda/write-cargo-toml.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+# Generates a Cargo.toml with the given AWS SDK version for this canary
+
+import argparse
+
+
+def main():
+    args = Args()
+
+    with open("Cargo.toml", "w") as file:
```

Comment:
seems like this might be a little clearer as a multiline string with string interpolation, (or are you trying to have this work on Python 2?)